docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
How about this note for the 1.8 release notes:
```
* Having an unmigrated app with a ``ForeignKey`` to a migrated app may now
  result in a foreign key constraint error (in Django 1.7, this could fail
  silently and result in a missing constraint). To resolve the error, add
  migrations to the unmigrated app.
```